CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+All notable changes to the Solr configuration for our Blacklight application will be documented in this file. The Solr configuration for our blacklight application contains the "Solr Schema" which is the basic configuration that defines the datatypes that will be ingested and queried.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and releases in blacklight_solr_conf project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Removed
+- Removed subtitle_t, subtitle_display, and subtitle_vern_display references [PR#5](https://github.com/ualbertalib/blacklight_solr_conf/pull/5)
```

Comment:
Might be worth adding a note about it's relationship with https://github.com/ualbertalib/discovery/pull/1596